.travis.yml

```diff
@@ -11,12 +11,21 @@ jobs:
       python: '3.6'
       install: pip install --upgrade black
       script: ./scripts/check-format
+      before_deploy: skip
+      deploy: skip
+      after_success: skip
+      after_failure: skip
     - stage: 'style checks'
       name: 'python lint'
       language: 'python'
       python: '3.6'
       install: pip install --upgrade pylint
+      deploy: skip
```

Comment:
Minor nit: is this `deploy` line extraneous given the one a few lines down?

Reply:
Yes, that is a typo, thanks!